exporter/logicmonitorexporter/testdata/config.yaml

```diff
@@ -0,0 +1,23 @@
+receivers:
```

Comment:
We do not recommend using full config for testing, see an example here https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/opencensusreceiver/testdata/config.yaml and you can check the usage https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/opencensusreceiver/config_test.go#L32

Reply:
Updated the PR taking reference from the above suggested example.